Sources/CocoaLumberjack/include/DDLog.h

```diff
@@ -850,6 +851,7 @@ typedef NS_OPTIONS(NSInteger, DDLogMessageOptions){
 @property (readonly, nonatomic) NSString *threadID; // ID as it appears in NSLog calculated from the machThreadID
 @property (readonly, nonatomic, nullable) NSString *threadName;
 @property (readonly, nonatomic) NSString *queueLabel;
+@property (readonly, nonatomic) NSUInteger qos;
```

Comment:
Should we mark this as available as well?

Reply:
Good point!

Reply:
See 3a35cc7